pkg/storage/utils/decomposedfs/node/node.go

```diff
@@ -125,6 +128,33 @@ func (n *Node) ChangeOwner(new *userpb.UserId) (err error) {
 	return
 }
 
+// SetMetadata populates a given key with its value.
+// Note that consumers should be aware of the metadata options on xattrs.go.
+func (n *Node) SetMetadata(key string, val string) (err error) {
```

Comment:
Is there any reason to bring those `[Set|Remove|Get]Metadata` methods back? They are bypassing the node's xattrs cache, potentially leading to inconsistent data.
I think we should use the `node.*Xattr` methods instead.

Reply:
Leftover from merge. Removed the `*Metadata` funcs and now use `*Xattr` instead